admin/templates/users/user.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block title %}
+    <title>User</title>
+{% endblock title %}
+{% block content %}
+    <div class="container-fluid">
+        <div class="row">
+            <form action="{% url 'users:user' %}" method="post">
+                {% csrf_token %}
+                {% if form.errors %}
+                    <div class="alert alert-danger">{{ form.errors }}</div>
+                {% endif %}
+                <div class="form-group">
+                    <label for="guid">User GUID:</label>
+                    {{ form.guid }}
+                </div>
+                <input type="submit" class="btn btn-primary" value="Submit" />
+            </form>
+        </div>
+
+        <hr>
+
+        {% if guid_object %}
+        {% with user=guid_object %}
+            <div class="row">
+                <h3>User information</h3>
+            </div>
+            <div class="row">
+                <strong>Name:</strong> {{ user.name }}
+            </div>
+            <div class="row">
+                <strong>Emails:</strong>
+                <ul>
+                {% for email in user.emails %}
+                    <li>
+                        {% if forloop.first %}
+                            <strong>Primary: </strong>
+                        {% endif %}
+                        {{ email }}
+                    </li>
+                {% endfor %}
+                </ul>
+            </div>
+            <div class="row">
+                <strong>Last logged in:</strong> {{ user.last_login }}
+            </div>
+            <div class="row">
+                <strong>Nodes</strong>
+                <table class="table table-striped table-hover table-responsive">
+                <thead>
+                    <tr>
+                        <th>GUID</th>
+                        <th>Name</th>
+                        <th>Public</th>
+                        <th># of Contributors</th>
+                    </tr>
+                </thead>
+                <tbody>
+                {% for node in user.nodes %}
+                    <tr>
+                        <td>
+                            <a href="{% url 'nodes:node' %}?guid={{ node.id }}">
```

Comment:
If you get to the point where you don't want to type this pattern out anymore django has the concept of both filters and template tags. https://docs.djangoproject.com/en/1.9/howto/custom-template-tags/#custom-template-tags-and-filters Either one would probably work fine in this circumstance, really up to you which you pick. Filters are normally used for smaller transmogrifying text type operations but that isn't a requirement.

Since this is practically the same as your success_url method on the view forms you could define a `reverse_node` and `reverse_user` that took an id and then write two wrapping functions and register them as filters. Or just register the functions as filters and have the success_url methods call those.

This is one of those things that won't really matter until you change your url pattern as you suggested in https://github.com/CenterForOpenScience/osf.io/pull/4960/files#r51434390 and then you are happy you did it.